src/solver/infeasible-problem-analysis/report.cpp

```diff
@@ -71,11 +72,8 @@ void InfeasibleProblemReport::sortConstraintsByType()
 {
     for (auto& c: constraints_)
     {
-        if (c.extractComponentsFromName() == 0)
```

Comment:
As already said above, **extractComponentsFromName** was returning the number of components a constraint's name is composed with, but this number can never be zero.
So the **if** statement was removed.